packages/jest-runtime/src/transform.js

```diff
@@ -288,7 +288,7 @@ const cachedTransformAndWrap = (
 const transformAndBuildScript = (
   filename: Path,
   config: Config,
-  options?: Options,
+  options: ?Options,
```

Comment:
cc @dmitriiabramov I'm unsure how this made it in. It's a babel and flow error locally (after doing `npm run clean; npm install`).